main.go

```diff
@@ -191,7 +191,6 @@ func registryCommand(ctx context.Context, args *RegistryCmd) error {
 	// Registry
 	registryOpts := []registry.RegistryOption{
 		registry.WithRegistryFilters(args.RegistryFilters),
-		registry.WithResolveRetries(args.MirrorResolveRetries),
```

Comment:
Why did you remove this? This change does not seem to be related to removing the latest check, and this means the `--mirror-resolve-retries` flag is no longer honored.